packages/elements/src/slider/elements/slider.ts

```diff
@@ -951,20 +951,29 @@ export class Slider extends FormFieldElement {
    */
   private updateNotifyProperty(name: SliderDataName, value: string): void {
     let shouldUpdate = false;
-
-    if (name === SliderDataName.to) {
-      shouldUpdate = this.isValueInBoundary(Number(value), SliderDataName.to);
-    } else {
-      shouldUpdate = this.isValueInBoundary(Number(value), '');
-    }
+    const valueNumber = Number(value);
+    shouldUpdate = this.isValueInBoundary(valueNumber, name);
+
+    // Calculate updated value based on the current slider value.
+    // From and to value is calculate with min range
+    const calculateValue = (currentValue: string): string => {
```

Comment:
* `adjustValue` should be a better name than `calculateValue`.
* keep it simple with `value` instead of `currentValue`?